core/ktor/src/main/java/ir/composenews/ktor/HttpClientExtension.kt

```diff
@@ -0,0 +1,17 @@
+package ir.composenews.ktor
+
+import io.ktor.client.HttpClient
+import io.ktor.client.request.HttpRequestBuilder
+import io.ktor.client.request.request
+import io.ktor.http.HttpMethod
+
+suspend inline fun <reified T> HttpClient.getApiResponse(
```

Comment:
If this method does ```GET``` request so we can rename it to ```get```

and we don't need ```ApiResponse``` postfix. because our context here is HttpClient and we are explicitly talking about GET request

Reply:
You're absolutely right. The suggestion is logical. Given that we're working within the context of HttpClient in Ktor, using the `ApiResponse` suffix seems redundant.

Also, this class can be easily expanded in the future to include more HTTP methods such as POST, PUT, and others, based on our project requirements